lib/backend/k8s/k8s_test.go

```diff
@@ -2420,7 +2420,6 @@ var _ = testutils.E2eDatastoreDescribe("Test Watch support", testutils.Datastore
 				case e := <-watch.ResultChan():
 					// Got an event. Check it's OK.
 					Expect(e.Error).NotTo(HaveOccurred())
-					Expect(e.Type).To(Equal(api.WatchAdded))
```

Comment:
Why this change?

Reply:
I need to dig a bit more here, but that test was failing with either `WatchAdded` or `WatchModified`

Reply:
Seems like a flake, it's passing now.